src/domUpdates.js

```diff
@@ -18,12 +19,16 @@ const domUpdates = {
      location.reload();
   }),
 
-  displayCustomerSearchResults: $('.select-date-button').on('click', () => {
-     hideOrShowElement('show', '.available-res-container');
-     hideOrShowElement('hide', '.book-a-cabin-container');
-     hideOrShowElement('hide', '.past-future-container');
-     hideOrShowElement('show', '.new-search-button');
-  }),
+  displayCustomerSearchResults: () => {
+    let user = giveUser();
+    user.searchDate = $('.date-input').val();
+    user.addAvailableCabinsToBookingSearch()
+    console.log(user.availableRooms);
```

Comment:
Don't forget to remove logs before submitting for review.

Reply:
Looks like I missed that one. Thanks for looking out! I'll make sure to look for those for my next PR.